proposals/0033-timely-vote-credits.md

```diff
@@ -0,0 +1,342 @@
+   VoteStateVersion will be behind a feature switch; until this feature is
+   enabled, validators will internally represent Vote accounts using the new
+   VoteStateVersion, but will always serialize and store vote account contents
+   using the prior version.  This does not lose information as the latency
+   value will not be used for anything before a later change, and thus is
+   implicitly always valued as '0', and on reading the vote account and
+   converting it to the newest VoteStateVersion, the default value of '0' for
+   latency will reconstitute the value of 0 that was in that field at the time
+   that the VoteState was serialized.
+
+   Once this software version has been integrated into the cluster and proven
+   for a period of time, would the feature be enabled.  At this point,
+   validators will begin writing vote account state using the newest
+   VoteStateVersion, and their snapshots will no longer be usable by the old
+   software versions, which by this point will be so old that rollback to them
+   is a practical impossibility anyway.
+
+   In addition, the vote state of an account cannot be updated to the new
+   larger form unless the vote account has sufficient lamports to cover the
+   new higher rent exempt minimum.  The new size of vote accounts will be 3762
+   bytes, a 31 byte increase on the old size of 3731 bytes.  The rent exempt
+   minimum for vote accounts will thus increase from 0.02685864 SOL to
+   0.0270744 SOL.  This is an additional 0.00021576 SOL per vote account.
+   Given roughly 2,000 vote accounts currently on mainline, the maximum SOL
+   needed to increase all vote accounts to the required new minimum is 0.43152
+   SOL.  The true value will certainly be less than this because some vote
+   accounts will already be above the new minimum.  In any case, the author of
+   the SIMD will send any additional SOL needed to increase rent exempt
+   minimum of all vote accounts, in the weeks leading up to the enablement of
+   this feature.  By the time the feature is enabled, the vast majority of
+   vote accounts will be funded to the new rent exempt minimum.
+
+   For any vote account not sufficiently funded, the old VoteState version
+   will be written.  Since the old VoteState does not record latencies, vote
+   account deserialization will assign the default value of 0 to the Lockout
+   latency field, which will result in the minimum vote credits awarded for
+   each vote for that vote account.  This provides economic incentive to
+   validators to sufficiently fund their vote accounts to allow the updated
+   VoteStateVersion to be serialized into them.
+
+   To summarize the issues surrounding the Vote account updates:
+
+   - A vote account will only be updated to the new version after the feature
+     is enabled, and only if the vote account includes enough SOL to satisfy
+     the new rent exempt minimum.  - If a vote account is not updated, then it
```

Comment:
is this intended to be 2 bullet points?

Reply:
Will fix, got messed up in recent auto-fill